assets/js/add-task.js

```diff
@@ -0,0 +1,78 @@
+window.addEventListener("load", () => {
+  const form = document.querySelector("#create-task");
+  const inputValue = document.querySelector("#input-value");
+  const taskList = document.querySelector("#task-list");
+  const createButton = document.querySelector("#create-button");
+  const cancelButton = document.querySelector("#cancel-button");
+
+  createButton.addEventListener("click", () => {
+    if (form.className == "hide") {
+      form.classList.remove("hide");
+    }
+    inputValue.focus();
+  });
+
+  cancelButton.addEventListener("click", () => {
```

Comment:
We may use const for the string "click"
`const CLICK_EVENT = "click";`

It would not have any significant impact on the performance of your code. However, using a constant may increase the readability of your code by making it easier to understand the purpose of the event listener.
**So, we may do this for all the event listeners.**